### PR TITLE
Remove tests.data.generated

### PR DESCRIPTION
**Pull request by breerly**
- [x] remove `tests.data.generated`
- [x] update tests

### CODE REVIEW

**Comment by breerly** [comment]
for #258

**Comment by abhinav** [comment]
:+1:

File: tests/serializer/test_serializer_thrift.py [hunk 1]
```diff
@@ -35,11 +31,3 @@ def test_header(v1):
     assert v1 == serializer.deserialize_header(
         serializer.serialize_header(v1)
     )
-
-
-def test_body():
```
Comment (breerly) [inline tests/serializer/test_serializer_thrift.py]: this test uses generated code and tests functionality that will be deleted in this release, so I axed it

**Comment by breerly** [comment]
@junchaowu @blampe good to go

**Comment by junchaowu** [comment]
lgtm

**Pull request merged**